netcode.c

```diff
@@ -3754,6 +3758,61 @@ void netcode_server_send_client_packet( struct netcode_server_t * server, void *
     server->client_last_packet_send_time[client_index] = server->time;
 }
 
+/**
+ * @brief skillz_print_all_matches
+ *
+ * Mostly just a test function.  Prints every match in the hash.
+ *
+ */
+void skillz_print_all_matches( struct netcode_server_t * server )
+{
+    skillz_match_t * match;
+
+    netcode_printf( NETCODE_LOG_LEVEL_INFO, "\n\nPrinting the matches and their clients.\n" );
+    int i;
+    for( match = server->skillz_matches; match != NULL; match = ( skillz_match_t * )( match->hh.next ) )
+    {
+        for( i = 0; i < match->num_clients_in_match; ++i)
```

Comment:
`for( int i` and you can avoid the declaration above

Reply:
I will change it, but it will be inconsistent with netcode.io syntax and older C compilers will yell at you.

Reply:
Ohhh C.... I'm ok with consistency here